gitops/components/messaging/rabbitmq/single-instance/README.md

```diff
@@ -0,0 +1,100 @@
+## RabbitMQ Single instance deploy spec
+
+### Prerequisite
+
+1. Have openshift credentials and have read/write permission
+2. Run the following commands under \gitops\components\messaging\rabbitmq folder.
+
+### Steps
+
+1. login openshift
+
+```
+oc login --token=yourtoken --server=https://api.silver.devops.gov.bc.ca:6443
+```
+
+2. go to the correct project
+
+```
+oc project 0198bb-dev
+```
+
+3. create rabbitmq-config config map
+
+```
+oc create -f config-map.yaml --save-config
+```
+
+> Note: in config-map.yaml, we remove rabbitmq_peer_discovery_k8s (which is default enabled) from enabled_plugins, as we only have 1 rabbitMQ instance. For details, please refer to [here](https://github.com/bitnami/charts/issues/2227).
+> We also add shovel plugin for error processing. For details, please refer to [here](https://www.rabbitmq.com/shovel.html).
+
+4. create rabbitmq secrets, which include Erlang-cookie, default username and password
+
+```
+oc create -f secrets.yaml --save-config
+```
+
+5. create secrets for docker access (dockerhub credentials for pull in dockerhub images)
+
+```
+oc create secret docker-registry docker-creds ^
+--docker-server=docker.io ^
+--docker-username=bcgovfams3 ^
+--docker-password="@&W9H0e#eLlHv5&Z@pR1t" ^
```

Comment:
Please redact and change the password changed ASAP

Reply:
changed